theme/layouts/_navigation.ejs

```diff
@@ -1,22 +1,54 @@
-<nav id="main-navigation">
+<%
+const query = site.queryPages;
+
+function getSubPages(nav)
+{
+  const subPages = query(({originalPathname}) => {
+    const parentPath = `${nav.originalPathname}/`;
+    return originalPathname.startsWith(parentPath) &&
+           !(originalPathname.substring(parentPath.length).includes("/"));
+  });
+  if (subPages.length) {
+    if (!nav.sub_items){
+      nav.sub_items = [];
+      for (const subPage of subPages) {
+        if(subPage.menu !== false) {
+        nav.sub_items.push(subPage);
+        getSubPages(subPage);
```

Comment:
Detail: Let's use space separators between `if` and `(subPage.menu !== false)` to make it consistent with other expressions here.

Detail2: indentation is missing in the `if` block.